### ISSUE TITLE
PNG sprite mode: rendering native emojis doesn't always work

### ISSUE THREAD

**Issue by DawidMyslak** [Ranks/emojione#432]
If you try to use native emojis (you can insert them with ctrl+cmd+space on Mac) in the PNG sprite mode (like this one for example: ❣️), police officer (:cop_tone2:) will be rendered instead. Cop is on position [0, 0] in the PNG sprite sheet. Obviously it's working fine for most of the emojis, but I found few that failed.

Here is the reason, "❣️" is being rendered as:
```
<span class="emojione emojione-2763-fe0f" title=":heavy_heart_exclamation_mark_ornament:">❣️</span>
```
Sprite file 2763-fe0f.png doesn't exist, so also CSS class `.emojione-2763-fe0f { ... }` is not generated. However, CSS class `emojione-2763` does exist and produces correct emoji.

**Comment by paulshapiro** [comment]
Yes, I'm experiencing this too. Definitely a big issue... would be very appreciative of a workaround. Thanks!!

**Comment by caseyahenson** [comment]
This is fixed with the release of 3.0.
